projects/question-editor-library/src/lib/services/question/question.service.ts

```diff
@@ -15,12 +15,14 @@ export class QuestionService {
     this.http = http;
   }
 
-  readQuestion(questionId) {
+  readQuestion(questionId, leafFormConfigfields? ) {
+    // tslint:disable-next-line:max-line-length
+    let field = 'body,primaryCategory,mimeType,qType,answer,templateId,responseDeclaration,interactionTypes,interactions,name,solutions,editorState,media,';
```

Comment:
@pallakartheekreddy  can we have this fields configured somewhere? cc: @santoshilimi

Reply:
Done